js/ui/paygUnlockDialog.js

```diff
@@ -0,0 +1,395 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+
+const Atk = imports.gi.Atk;
+const Clutter = imports.gi.Clutter;
+const Gio = imports.gi.Gio;
+const GLib = imports.gi.GLib;
+const GObject = imports.gi.GObject;
+const Lang = imports.lang;
+const Meta = imports.gi.Meta;
+const Pango = imports.gi.Pango;
+const Signals = imports.signals;
+const Shell = imports.gi.Shell;
+const St = imports.gi.St;
+
+const PaygManager = imports.misc.paygManager;
+
+const Animation = imports.ui.animation;
+const Main = imports.ui.main;
+const Monitor = imports.ui.monitor;
+const ShellEntry = imports.ui.shellEntry;
+const Tweener = imports.ui.tweener;
+
+// The timeout before going back automatically to the lock screen (in seconds)
+const IDLE_TIMEOUT = 2 * 60;
+
+const CODE_ENTRY_REQUIRED_LENGTH = 8;
+
+var SPINNER_ICON_SIZE = 16;
+var SPINNER_ANIMATION_DELAY = 1.0;
+var SPINNER_ANIMATION_TIME = 0.3;
+
+var MESSAGE_FADE_OUT_ANIMATION_TIME = 0.5;
```

Comment:
Nitpick: Same for all the above.

Reply:
Ok. Also, I'll remove the last one, which is unused (comes from unlockDialog.js)